typespec-extension/src/code-model-builder.ts

```diff
@@ -429,6 +434,14 @@ export class CodeModelBuilder {
     }
   }
 
+  private needToSkipProcessingOperation(operation: Operation): boolean {
+    // don't generate protocol and convenience method for overloaded operations
```

Comment:
It might be good to add additional details on why we are skipping this or link to a GitHub issue where we've discussed this.